src/store/index.js

```diff
@@ -1,21 +1,39 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from 'vue';
+import Vuex from 'vuex';
 
-import user from './modules/user'
+import user from './modules/user';
+import contacts from './modules/contacts';
 
-Vue.use(Vuex)
+Vue.use(Vuex);
 
 export default new Vuex.Store({
   state: () => ({
-    serverAddress: 'https://phonebook-api.laus.me'
+    serverAddress: 'https://phonebook-api.laus.me',
+    alertNotification: {
+      status: false,
+      color: 'error',
+      text: 'Unhandled error.'
+    }
   }),
   getters: {
+    alertNotification(state) {
+      return state.alertNotification;
+    },
   },
   mutations: {
+    createNewAlert(state, payload) {
+      state.alertNotification.text = payload.text;
+      state.alertNotification.color = payload.color;
+      state.alertNotification.status = true;
+    },
+    setAlertStatus(state, payload) {
+      state.alertNotification.status = payload;
+    }
   },
   actions: {
   },
   modules: {
-    user
+    user,
+    contacts
```

Comment:
Also there can be used contact, instead of contacts, due to singular meaning of module name.